src/webgl/light.js

```diff
@@ -284,4 +284,35 @@ p5.prototype.pointLight = function(v1, v2, v3, x, y, z) {
   return this;
 };
 
+/**
+ * Sets a default value for ambient and directional light
```

Comment:
Just a quick documentation edit request - I feel like this is a good opportunity to show what the code is doing just a bit more. The first couple of lines from the Processing documentation for `lights` does this well with "The defaults are ambientLight(128, 128, 128) and directionalLight(128, 128, 128, 0, 0, -1).

Reply:
Good idea, I used the description in [Processing's docs](https://processing.org/reference/lights_.html) as a model.